### PR TITLE
Moon 36: Add active state to MenuItem

### PR DESCRIPTION
**Pull request by ffffffelix**
## JIRA

https://jira.jahia.org/browse/MOON-36

## Description
- Remove all states from `ListItem`, now all states are managed from `MenuItem`
- Add active state on `MenuItem`
- Allow opening the dropdown by pressing Enter when it's focused

## Tests

The following are included in this PR

- [ ] Unit Test skeleton

## Checklist

- [ ] All files present ( component - md - scss - spec - stories )
- [ ] Are all props ok and documented
- [ ] Required props and default values
- [ ] Example in storybook

## Documentation

- [X] README documentation

### CODE REVIEW

File: src/components/Dropdown/Dropdown.md [hunk 1]
```diff
@@ -9,3 +9,8 @@ Allow user to validate a value by pressing `Enter`
 To hide the menu click on an item of the list or click outside to it
 The selected item has a specific design
 Items can be grouped with a title
+
+
+## Implementation
+
+To allow to use the dropdown with keyboard don't forget to add `e.stopPropagation();` on the onChange callback
```
Comment (tdraier) [inline src/components/Dropdown/Dropdown.md]: Shouldn't it be done directly in DropDown component ?

Comment (ffffffelix) [reply src/components/Dropdown/Dropdown.md]: Yes, it should be better, but I don't find the right place to add it without break everything

Comment (ffffffelix) [reply src/components/Dropdown/Dropdown.md]: I find a way 🤘

**Comment by ffffffelix** [comment]
> And so we keep the nice colors ? :-)

I use colors from Figma

**Pull request merged**